client/src/d3.js

```diff
@@ -0,0 +1,65 @@
+import React, { Component } from 'react';
+import { scaleLinear, scaleBand } from 'd3-scale';
+import { select } from 'd3-selection';
+import { axisBottom, axisLeft } from 'd3-axis';
+import { max } from 'd3-array';
+
+class D3 extends Component {
```

Comment:
Where is this used?

Reply:
again, comment with a todo to delete this later (this way when you are polishing things up later you can just look for todos and deal with them then).